OutOfSchool/OutOfSchool.DataAccess/Models/Workshop.cs

```diff
@@ -56,6 +56,9 @@ public class Workshop : IKeyedEntity<Guid>, IImageDependentEntity<Workshop>
     [Range(0, 18, ErrorMessage = "Max age should be a number from 0 to 18")]
     public int MaxAge { get; set; }
 
+    [Required(ErrorMessage = "CompetitiveSelection is required")]
+    public bool CompetitiveSelection { get; set; } = default;
```

Comment:
What is the purpose of setting `default` as the initial value here?
```suggestion
    [Required(ErrorMessage = "CompetitiveSelection is required")]
    public bool CompetitiveSelection { get; set; }
```

Reply:
fixed.